sdk/servicebus/service-bus/package.json

```diff
@@ -44,7 +44,7 @@
     "@types/debug": "^0.0.31",
     "@types/dotenv": "^6.1.0",
     "@types/mocha": "^5.2.5",
-    "@types/node": "^8.0.37",
+    "@types/node": "^8.0.0",
```

Comment:
Why did you change this version?

Reply:
![image](https://user-images.githubusercontent.com/10452642/57050991-0afd9180-6c34-11e9-9617-56f0d55095aa.png)
"rush update" shows "mismatching dependencies"

Reply:
It's fine for this PR, but in the future I would split this into a separate PR since it's not related to the new test.  It's easier to review and understand PRs when they are scoped to a single logical change.